subsys/bluetooth/controller/ll_sw/ull_sync_types.h

```diff
@@ -8,6 +8,22 @@
 #define LL_SYNC_STATE_ADDR_MATCH 0x01
 #define LL_SYNC_STATE_CREATED    0x02
 
+#if defined(CONFIG_BT_CTLR_SYNC_ISO)
+struct ll_sync_iso {
+	struct evt_hdr evt;
+	struct ull_hdr ull;
+	struct lll_sync_iso lll;
+
+	struct node_rx_hdr node_rx_lost;
```

Comment:
This does not seem to be really used now. I assume it will be used eventually to create BIG Sync Lost events?

Reply:
yes